test/e2e/apimachinery/webhook.go

```diff
@@ -1126,11 +1128,12 @@ func registerValidatingWebhookForWebhookConfigurations(f *framework.Framework, c
 				Rules: []admissionregistrationv1.RuleWithOperations{{
 					Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.Delete},
 					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{"admissionregistration.k8s.io"},
+						APIGroups:   []string{"", "admissionregistration.k8s.io"},
```

Comment:
separate rule (apiGroup:"" doesn't make sense for validatingwebhookconfigurations/mutatingwebhookconfigurations)